### PR TITLE
Service bus: Added 2014 swagger spec

### PR DESCRIPTION
**Pull request by v-Ajnava**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json [hunk 1]
```diff
@@ -0,0 +1,2620 @@
...
+        },
+        "secondaryKey": {
+          "type": "string",
+          "description": "A base64-encoded 256-bit primary key for signing and validating the SAS token."
+        },
+        "keyName": {
+          "type": "string",
+          "description": "A string that describes the authorization rule."
+        },
+        "claimType": {
```
Comment (alvadb) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: The description appears to be incorrect.

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json [hunk 2]
```diff
@@ -0,0 +1,2620 @@
...
+          "type": "string",
+          "description": "A string that describes the authorization rule."
+        },
+        "claimValue": {
```
Comment (alvadb) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: The description appears to be incorrect.

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json [hunk 3]
```diff
@@ -0,0 +1,2620 @@
...
+          "format": "int64",
+          "type": "integer",
+          "readOnly": true,
+          "description": "Number of scheduled messages."
+        },
+        "transferDeadLetterMessageCount": {
+          "format": "int64",
+          "type": "integer",
+          "readOnly": true,
+          "desc
```
Comment (alvadb) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: The description references topic but I believe should reference subscription.

Comment (alvadb) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: Need to upper case the p in properties

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json [hunk 4]
```diff
@@ -0,0 +1,2620 @@
...
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Namespace authorization rule created.",
```
Comment (alvadb) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: If this is for updated then the description should say rule updated and not create.

**Review by alvadb — CHANGES_REQUESTED** [review]

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json (hunk 4 above)
Comment (v-Ajnava) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: resolved

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json (hunk 1 above)
Comment (v-Ajnava) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: corrected

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json (hunk 2 above)
Comment (v-Ajnava) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: Corrected

File: specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json (hunk 3 above)
Comment (v-Ajnava) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: MessageCountDetails, is refereed in queue, topic and subscription

Comment (v-Ajnava) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: resolved

Comment (v-Ajnava) [inline specification/servicebus/resource-manager/Microsoft.ServiceBus/2014-09-01/servicebus.json]: reverted the change as it created syntax error

**Review by alvadb — APPROVED** [review]

**Pull request merged**